README.md

```diff
@@ -6,6 +6,10 @@ Callstack ESLint config utilizing Flow/TypeScript, Prettier, Jest, React and Rea
 
 Plugins and configs used:
 
+- React Native
+  - Default config
+  - [eslint-plugin-react-native-a11y](https://classic.yarnpkg.com/en/package/eslint-plugin-react-native-a11y)
```

Comment:
Can we remove this addition on only add the `eslint-plugin-react-native-a11y` to the "Default config" list? also, make sure it links to https://yarnpkg.com instead of https://classic.yarnpkg.com, so it's consistent with other links